### PR TITLE
Cleanup formatting ocp4_workload_example* roles

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Added yamllint files to the roles
- Changed formatting as required for the files in the roles
- Made warnings into errors in yamllint config for future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp4_workload_example
ocp4_workload_example_dedicated_cluster
ocp4_workload_example_shared_cluster

### CODE REVIEW

File: ansible/roles_ocp_workloads/ocp4_workload_example/tasks/post_workload.yml [hunk 1]
```diff
@@ -26,4 +24,4 @@
     msg: "Post-Software checks completed successfully"
   when:
   - not silent|bool
-  - workload_shared_deployment|default(False)
\ No newline at end of file
+  - workload_shared_deployment|default(False)
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example/tasks/post_workload.yml:27]: This should be:

```
  - workload_shared_deployment | default(false) | bool
```

Not something the yamllint would catch, but will come up with ansible linting

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/post_workload.yml [hunk 2]
```diff
@@ -26,4 +24,4 @@
     msg: "Post-Software checks completed successfully"
   when:
   - not silent|bool
-  - workload_shared_deployment|default(False)
\ No newline at end of file
+  - workload_shared_deployment|default(False)
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/post_workload.yml:27]: Same here:
```
  - workload_shared_deployment | default(false) | bool
```

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml [hunk 3]
```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
+   msg: "Remove Workload tasks completed successfully."
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml:16]: This doesn't look right... looks like it went from two space indent to one?

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/workload.yml [hunk 4]
```diff
@@ -33,7 +33,9 @@
   until:
     - r_hello_openshift_deployment.resources | length | int > 0
     - r_hello_openshift_deployment.resources[0].status.readyReplicas is defined
-    - r_hello_openshift_deployment.resources[0].status.readyReplicas | int == r_hello_openshift_deployment.resources[0].spec.readyReplicas | int
+    - >-
+      r_hello_openshift_deployment.resources[0].status.readyReplicas
+      | int == r_hello_openshift_deployment.resources[0].spec.readyReplicas | int
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/workload.yml:38]: Probably would read better with that `| int` filter on the previous line:
```
      r_hello_openshift_deployment.resources[0].status.readyReplicas | int
      == r_hello_openshift_deployment.resources[0].spec.readyReplicas | int
```

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/post_workload.yml (hunk 2 above)
Comment (ritesh97-rh) [reply ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/post_workload.yml:27]: Haven't changed things like that anywhere. Will look over this and make the changes.

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml (hunk 3 above)
Comment (ritesh97-rh) [reply ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml:16]: Yeah.. its what the formatter wanted lol

File: ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml [hunk 5]
```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
+   msg: "Remove Workload tasks completed successfully."
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml:16]: Spacing looks wrong here.

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml (hunk 3 above)
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml:16]: Indent looks wrong

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/workload.yml (hunk 4 above)
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/workload.yml:38]: `| int` filter should be on previous line.

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/files/example_app_template.yml [hunk 6]
```diff
@@ -1,12 +1,16 @@
...
 apiVersion: template.openshift.io/v1
 kind: Template
 labels:
   app: httpd-example
   template: httpd-example
+# yamllint disable rule:line-length
 message: |-
   The following service(s) have been created in your project: ${NAME}.
 
   For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/httpd-ex/blob/master/README.md.
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/files/example_app_template.yml:11]: We can meet the line length rule with:
```
message: >-
  The following service(s) have been created in your project: ${NAME}.


  For more information about using this template, including OpenShift considerations,
  see https://github.com/sclorg/httpd-ex/blob/master/README.md.
```

File: ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml [hunk 7]
```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
-  when: not silent|bool
+   msg: "Remove Workload tasks completed successfully."
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml:16]: Removed space here?

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml [hunk 8]
```diff
@@ -13,5 +13,5 @@
 # --------------------------------------------
 - name: remove_workload tasks complete
   debug:
-    msg: "Remove Workload tasks completed successfully."
-  when: not silent|bool
+   msg: "Remove Workload tasks completed successfully."
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml:16]: Removed space?

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml (hunk 7 above)
Comment (ritesh97-rh) [reply ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/tasks/remove_workload.yml:16]: yamllint wants indent 3 at that spot. It gives an error if I leave the space.

File: ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml (hunk 8 above)
Comment (ritesh97-rh) [reply ansible/roles_ocp_workloads/ocp4_workload_example_shared_cluster/tasks/remove_workload.yml:16]: yamllint wants indent 3 at that spot. If I leave the space there, it gives me an error.

File: ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/files/example_app_template.yml [hunk 9]
```diff
@@ -1,12 +1,15 @@
...
   app: httpd-example
   template: httpd-example
-message: |-
+message: >-
   The following service(s) have been created in your project: ${NAME}.
 
-  For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/httpd-ex/blob/master/README.md.
+  For more information about using this template, including OpenShift considerations,
+  see https://github.com/sclorg/httpd-ex/blob/master/README.md.
+
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/files/example_app_template.yml:12]: This is not equivalent as it drops a newline.

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**